proto/buf/alpha/registry/v1alpha1/search.proto

```diff
@@ -16,6 +16,16 @@ syntax = "proto3";
 
 package buf.alpha.registry.v1alpha1;
 
+enum SearchFilter {
```

Comment:
I think others might have different opinions, but I think this is a great use of enums to drive search filtering.

Afaics we don't normally use enums on the request side of the RPC, so worth calling out to see if anyone has major objections?

Reply:
yeah for sure going to circulate! still have this in draft so i was going to wait for more comments on my other pr about our approach before i start propagating it into requests 😸